analysis/alignment/requantAlignedValues.py

```diff
@@ -928,10 +927,16 @@ def _integrate_chromatogram(template_pg, current_run, swath_chromatograms,
 
     for chrom_id in chrom_ids:
         peaks = swath_chromatograms.getChromatogram(current_rid, chrom_id)
+
         # Check if we got a chromatogram with at least 1 peak:
         if peaks is None:
-            print("chromatogram is None (tried to get %s from run %s)" % (chrom_id, current_rid))
-            return("NA")
+            # pyprophet export files do not contain the actual chromatogram id
+            # written in mzML / sqMass but they contain "id_ion_ordinal" from
+            # which we can parse the id
+            peaks = swath_chromatograms.getChromatogram(current_rid, chrom_id.split("_")[0])
```

Comment:
see https://github.com/PyProphet/pyprophet/issues/48

Reply:
Yes, this is a matter of compatibility with the legacy format. Maybe we could just add an additional column.